src/views/viewer/DiscoverView.tsx

```diff
@@ -0,0 +1,235 @@
+  },
+  'news-and-politics': {
+    id: '11',
+    title: 'News & Politics',
+    icon: <SvgVideoCategoriesNewsAndPolitics />,
+    color: '#48F0B3',
+    coverImg: 'https://eu-central-1.linodeobjects.com/atlas-assets/category-images/news-and-politics.webp',
+  },
+} as const
+const featuredVideoCategories: Array<{ videoUrl: string; videoTitle: string } & VideoCategoryData> = [
+  {
+    ...videoCategories['travel-and-events'],
+    videoUrl: 'https://sumer-dev-2.joystream.app/storage/asset/v0/5Fbef6KfEP3ncHxroVsdWQF6gLb8ph47dcAmzWptjuMMWHnP',
+    videoTitle: 'Anderson .Paak and The Free Nationals',
+  },
+  {
+    ...videoCategories['education'],
+    videoUrl: 'https://sumer-dev-2.joystream.app/storage/asset/v0/5Fbef6KfEP3ncHxroVsdWQF6gLb8ph47dcAmzWptjuMMWHnP',
+    videoTitle: 'Anderson .Paak and The Free Nationals',
+  },
+  {
+    ...videoCategories['film-and-animation'],
+    videoUrl: 'https://sumer-dev-2.joystream.app/storage/asset/v0/5Fbef6KfEP3ncHxroVsdWQF6gLb8ph47dcAmzWptjuMMWHnP',
+    videoTitle: 'Anderson .Paak and The Free Nationals',
+  },
+]
+
+export const DiscoverView: React.FC = () => {
+  const { videosConnection } = useVideosConnection(
+    {
+      first: 0,
+    }
+    // { skip: !anyFollowedChannels, onError: (error) => SentryLogger.error('Failed to fetch videos', 'HomeView', error) }
+  )
+  const isMdBreakpoint = useMediaMatch('md')
+  return (
+    <StyledLimitedWidthContainer big>
+      <Text variant="h2">Discover</Text>
+      <FeaturedCategoriesContainer>
+        {featuredVideoCategories.map((category, i) => (
+          <GridItem key={i} colSpan={{ sm: i === 0 ? 2 : 1, xl: 1 }}>
+            <FeaturedVideoCategoryCard
+              variant={isMdBreakpoint ? 'default' : 'compact'}
+              title={category.title}
+              videoTitle={category.videoTitle}
+              videoUrl={category.videoUrl}
+              color={category.color}
+              icon={category.icon}
+            />
+          </GridItem>
+        ))}
+      </FeaturedCategoriesContainer>
+      <BorderTextContainer>
+        <Text variant="h4">All categories</Text>
+      </BorderTextContainer>
+      <CategoriesContainer>
+        {Object.values(videoCategories).map((category, i) => (
+          <VideoCategoryCard
+            key={i}
+            title={category.title}
+            coverImg={category.coverImg}
+            categoryId={category.id}
+            color={category.color}
+            icon={category.icon}
+            videosTotalCount={videosConnection?.totalCount}
+            variant={isMdBreakpoint ? 'default' : 'compact'}
+          />
+        ))}
+      </CategoriesContainer>
+    </StyledLimitedWidthContainer>
+  )
+}
+
+const StyledLimitedWidthContainer = styled(LimitedWidthContainer)`
+  margin: ${sizes(16)} auto;
+`
+
+const FeaturedCategoriesContainer = styled.div`
+  display: grid;
+  gap: ${sizes(4)};
```

Comment:
```suggestion
  gap: ${sizes(6)} ${sizes(7)};
```
design: 
![image](https://user-images.githubusercontent.com/51168865/131837258-e31e5476-9568-4128-9ede-7bc97ffba277.png)

![image](https://user-images.githubusercontent.com/51168865/131837302-019f857e-bf54-4191-a760-e1cf2735f294.png)

Reply:
it's just `gap: ${sizes(6)}` if you look closely you had the horizontal measurement over the cards border that is 4px wide

Reply:
My mistake, you're right 👍🏻

Reply:
This discussion was resolved but the code is still old?